### ISSUE TITLE
msixmgr does not load package resouces

### ISSUE THREAD

**Issue by GITNE** [microsoft/msix-packaging#230]
*Project*
MSIXCore msixmgr

**Describe the bug**
msixmgr does not resolve `ms-resource` protocol references.

**To Reproduce**
Create a package manifest with `ms-resource` protocol references where applicable ([localizable manifest items](https://docs.microsoft.com/en-us/uwp/schemas/appxpackage/uapmanifestschema/localizable-manifest-items-win10)). Observe the install UI.

**Expected behavior**
msixmgr should load resources from packages.

**Screenshots**
![NoLocalizableMSIXManifestElements](https://user-images.githubusercontent.com/15823872/66259580-db3e5100-e7b2-11e9-9aa5-d801e960dfef.png)

**Platform**
Windows 7 and later

**Comment by jvintzel** [comment]
The downlevel operating systems do not support Modern Resource Technology which is required.  We do have a backlog item to look at providing a similar level of functionality for this.

**Comment by GITNE** [comment]
> We do have a backlog item to look at providing a similar level of functionality for this.

Right, my understanding is that for the purpose of MSIX Core we do not need a full implementation of the Resource Management System, i.e. the [`ResourceManager`](https://docs.microsoft.com/en-us/uwp/api/windows.applicationmodel.resources.core.resourcemanager) class. It should suffice to be able to resolve and load string and image resources. In fact, `MakePRI.exe` which also works on systems which lack the RMS should already implement a great deal of resource writing and possibly reading functionality, including resource resolution which is a complex core feature of the RMS.

Anyhow, thank you for taking this into consideration.
